tests/framework/config/ray-cluster.mini.yaml.template

```diff
@@ -8,65 +8,41 @@ metadata:
 spec:
   rayVersion: '$ray_version' # should match the Ray version in the image of the containers
   ######################headGroupSpecs#################################
-  # head group template and specs, (perhaps 'group' is not needed in the name)
+  # Ray head template
   headGroupSpec:
     # Kubernetes Service Type, valid values are 'ClusterIP', 'NodePort' and 'LoadBalancer'
     serviceType: ClusterIP
     # the pod replicas in this group typed head (assuming there could be more than 1 in the future)
     replicas: 1
-    # the following params are used to complete the ray start: ray start --head --block --redis-port=6379 ...
+    # the following params are used to complete the ray start: ray start --head --block --dashboard-host: '0.0.0.0' ...
     rayStartParams:
-      port: '6379' # should match headService targetPort
-      #include_webui: 'true'
-      redis-password: 'LetMeInRay' # Deprecated since Ray 1.11 due to GCS bootstrapping enabled
-      # webui_host: "10.1.2.60"
       dashboard-host: '0.0.0.0'
-      num-cpus: '1' # can be auto-completed from the limits
-      node-ip-address: $$MY_POD_IP # auto-completed as the head pod IP
```

Comment:
`node-ip-address: $MY_POD_IP` . Without this config, ray core will use pod id (node ip inside container)? This should be out of box supported?

Reply:
The ip is correctly resolved by the Ray core out of the box.